manual/source/Infrastructure/Manage/RepositoryStructure.rst

```diff
@@ -26,7 +26,7 @@ SeqAn supports the usual CMake build types and we recommend :ref:`using multiple
 In addition to ``CMAKE_BUILD_TYPE`` there is also the ``SEQAN_BUILD_SYSTEM`` parameter which can be one of
 
 #. ``DEVELOP`` -- all build targets (apps, demos, tests) and documentation (dox, manual) are created (the default).
-#. ``SEQAN_RELEASE_LIBRARY`` -- only dox and libraray target created.
+#. ``SEQAN_RELEASE_LIBRARY`` -- only dox and library targets are created.
```

Comment:
Oops. I found one thing.
According to #1560 "SEQAN_RELEASE_LIBRARY" will not contain dox anymore.
I didn't check that PR in detail but it describes like that.

Don't you think that we have to change this line?